Firestore/Swift/Source/SwiftAPI/PipelineResult.swift

```diff
@@ -0,0 +1,43 @@
+// swift-tools-version:5.9
```

Comment:
I thought setting `swift-tools-version` is only meaningful for Package.swift files? What does it mean here?